images/rootfs-mini.yml.in.patch

```diff
@@ -30,10 +31,10 @@
 -      - CAP_NET_ADMIN
 -  - name: modprobe
 -    image: linuxkit/modprobe:v0.5
--    command: ["/bin/sh", "-c", "modprobe -a nct6775 w83627hf_wdt hpwdt wlcore_sdio wl18xx br_netfilter dwc3 rk808 rk808-regulator smsc75xx cp210x nicvf tpm_tis_spi rtc_rx8010 gpio_pca953x leds_siemens_ipc127 upboard-fpga pinctrl-upboard leds-upboard xhci_tegra 2>/dev/null || :"]
-   # If you change the order of storage-init don't forget to
-   # change 003-installer in pkg/mkimage-raw-efi accordingly:
-   # storage-init must follow installer
+-    command: ["/bin/sh", "-c", "modprobe -a nct6775 w83627hf_wdt hpwdt wlcore_sdio wl18xx br_netfilter dwc3 rk808 rk808-regulator smsc75xx cp210x nicvf tpm_tis_spi rtc_rx8010 os-em32x regmap-i2c gpio_pca953x leds_gpio leds_siemens_ipc127 upboard-fpga pinctrl-upboard leds-upboard xhci_tegra 2>/dev/null || :"]
```

Comment:
I think only this change is needed, right?

Reply:
yes, but patch files are generated using YQ-processed yaml so some minor things are modified as well like redundant " or '

Reply:
It looks like you remove comment before storage-init as well (lines are prefixed by minus). I think it is not the problem, but curious, is it possible to leave the comment here in general?

Reply:
GO version of YQ can preserve comments but since I'm going to completely get rid of patch files this is not a problem because we do not care about comments in intermediate files